web/src/components/FileBrowser/FileUploadInstructions.vue

```diff
@@ -24,9 +24,9 @@
           >
             <div>{{ downloadCommand }}</div>
             <div>> cd {{ dandisetIdentifier }}</div>
-            <div>> lincbrain organize &lt;source_folder&gt; -f dry</div>
-            <div>> lincbrain organize &lt;source_folder&gt;</div>
-            <div>> lincbrain upload</div>
+            <div>> dandi organize -i linc &lt;source_folder&gt; -f dry</div>
+            <div>> dandi organize -i linc &lt;source_folder&gt;</div>
```

Comment:
I don't think that `-i` is an option for [dandi organize](https://dandi.readthedocs.io/en/latest/cmdline/organize.html).
```suggestion
            <div>> dandi organize &lt;source_folder&gt; -f dry</div>
            <div>> dandi organize &lt;source_folder&gt;</div>
```

Reply:
Updated!